golem-worker-service-base/src/service/api_definition.rs

```diff
@@ -191,33 +177,27 @@ where
                 return Err(ValidationError { errors }.into());
             }
 
-            successes
-                .into_iter()
-                .map(|r| r.unwrap())
-                .map(|t| t.value)
-                .collect()
+            successes.into_iter().map(|r| r.unwrap()).collect()
         };
 
         Ok(templates)
     }
 }
 
 #[async_trait]
-impl<AuthCtx, ApiDefNamespace, TemplateNamespace> ApiDefinitionService<AuthCtx, ApiDefNamespace>
-    for RegisterApiDefinitionDefault<AuthCtx, ApiDefNamespace, TemplateNamespace>
+impl<AuthCtx, Namespace> ApiDefinitionService<AuthCtx, Namespace>
+    for RegisterApiDefinitionDefault<AuthCtx, Namespace>
 where
     AuthCtx: Send + Sync,
-    ApiDefNamespace: ApiNamespace,
-    TemplateNamespace: std::fmt::Debug + Send + Sync,
+    Namespace: ApiNamespace + Send + Sync,
 {
     async fn register(
         &self,
         definition: &ApiDefinition,
-        auth_ctx: AuthCtx,
-    ) -> ApiResult<ApiDefinitionId, ApiDefNamespace> {
-        let namespace = self.is_authorized(Permission::Create, &auth_ctx).await?;
```

Comment:
Why is this deleted from ApiDefinitionService?

Reply:
the diff looks confusing. AuthCtx is still there